fedimint-core/src/module/mod.rs

```diff
@@ -292,6 +292,7 @@ pub trait IModuleGen: Debug {
     fn trusted_dealer_gen(
         &self,
         peers: &[PeerId],
+        module_id: ModuleInstanceId,
```

Comment:
Modules were not supposed to touch their ids (and be oblivious to them, just in case).

Reply:
But we pass it into `distributed_config_gen` for each module?

Reply:
Yes. That what `ConfigGenParams` was supposed to be. (see other comment)